src/vsphere_cpi/lib/cloud/vsphere/resources/storage_pod.rb

```diff
@@ -42,6 +56,12 @@ def self.find_storage_pod(name, datacenter_mob)
       def datastores
         mob.child_entity
       end
+
+      # @return [String] debug storagePod information.
+      def inspect
+        "<StoragePod: #@mob / #{name}>"
```

Comment:
Did you want to print the mob here? #{mob} ?

Reply:
This is the usual way every resource inspect has been implemented. So, I followed the convention. I do not know why it is done this way. 
I think it does give us a clue on the mob name and makes it easier to find in managed object browser of vsphere.

I will let it be for now